<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogoCaracteristicas</class>
 <widget class="QDialog" name="DialogoCaracteristicas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>511</width>
     <height>211</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>509</width>
      <height>209</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Imagen Original</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignTop</set>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="ImagenOriginalLB">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>128</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>130</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Imagen Preprocesada</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="ImagenPreprocesadaLB">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>128</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>130</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>311</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Relacion Ancho/Alto Rectangulo envolvente:</string>
   </property>
  </widget>
  <widget class="QLabel" name="RelacionLB">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>380</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="CalcularCaracteristicasBT">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Calcular Caracteristicas</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>581</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>End-Points</string>
   </property>
   <widget class="QLabel" name="EndPointsLB">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero de End-Points:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ImagenEndPointsLB">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>130</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccionar Caracter</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
